<launch>

  <include file="$(find jsk_topic_tools)/launch/topic_buffer_server_sample.launch" />

  <node pkg="roscpp_tutorials" type="listener" name="listener"
	output="screen">
    <remap from="chatter" to="chatter_buffered"/>
  </node>


  <node pkg="jsk_topic_tools" type="topic_buffer_client" name="sample_topic_buffer_client"
	output="screen">
    <remap from="/list" to="/sample_topic_buffer_server/list"/>
    <remap from="/update" to="/sample_topic_buffer_server/update"/>
    <param name="update_rate" value="0.1" />
    <param name="fixed_rate" value="0.01" />
  </node>

</launch>